packages/cli/commands/process.js

```diff
@@ -15,7 +15,7 @@ const { trackProcessFieldsUsage } = require('../lib/usageTracking');
 const i18nKey = 'cli.commands.process';
 
 exports.command = 'process';
```

Comment:
Thoughts on renaming this top level command to something more specific like `process-fields`?

@TanyaScales @brandenrodgers

Reply:
I agree it is more descriptive.

Reply:
We have a `hs module` command bucket. This seems like it could go in there, rather than live at the top level. I also agree that `process` is too vague. `process-fields` is definitely better IMO, but it's still not clear that it is referring to module fields.

`hs module process-fields` seems reasonable. What do you think about that?

Reply:
@brandenrodgers good call, I agree bucketing this would be better but this command also works for themes fields as well.

We could start a general `hs cms` bucket. Or actually I think you could make the case that whether you use this command for a theme or a module, it could be bucketed under `hs theme` since themes are the "parent feature" that you're using here.

Reply:
I'd almost ask use to reconsider the term `process` altogether -- It doesn't really explain what is going on, IMO. Using something like `convert` or `compile` is what comes to mind instead?

As to where it would sit -- we have what appears to be a top-level `cms` bucket already containing the `lighthouseScore` command. But, what comes to mind for me is a top level `fields` bucket instead, as there may be more commands in the future related to manipulating fields.json or fields.js files within themes/modules, and elsewhere.

So the end result would actually be `hs fields compile` or `hs fields compileJs` or something like that.

Reply:
I'd be +1 for 'convert` since it feels like that's closest to what we're doing.

I believe the ultimate goal is to consolidate commands into buckets so I'd be hesitant to introduce another top level `fields` bucket, especially since it would only have one command in it. I missed the `cms` bucket, though... `hs cms convertFields` seem good?

Reply:
We didn't get the change to talk about this specific case in the CLI sync today -- but I like the suggestion you have above @anthmatic and it aligns with what @brandenrodgers was talking about in the meeting (bucketing under `cms`). So, I'm in favor of the `hs cms convert-fields` command and flag as you have it laid out here.